easy-random-core/src/test/java/org/jeasy/random/EasyRandomTest.java

```diff
@@ -231,6 +231,7 @@ void nextEnumShouldNotAlwaysReturnTheSameValue() {
     }
 
     @Test
+    // this test fails on jdk17
```

Comment:
Thank you for raising this, I also noticed that. I created #494 to investigate the issue.